storagenode/orders/db_test.go

```diff
@@ -126,6 +126,15 @@ func TestDB(t *testing.T) {
 			},
 		}, archived, cmp.Comparer(pb.Equal)))
 
+		// with 1 hour ttl, archived order should not be deleted
```

Comment:
Could you help me understand why with 1 hour ttl, the order shouldn't be deleted?

Reply:
1 hour ttl means we want to delete all items that are older than 1 hour. Since the item was just inserted, it hasn't reached the 1 hour limit yet